test/torchaudio_unittest/io/stream_reader_test.py

```diff
@@ -360,6 +360,20 @@ def test_seek_negative(self):
             s.seek(-1.0)
 
 
+def _to_fltp(original):
```

Comment:
can you add doc string here? This helps future maintainer understand what's the functionality of the method.

Reply:
sure, done.